draft-ietf-jsonpath-base.md

```diff
@@ -1010,40 +1007,44 @@ The current item is selected if and only if the result is `true`.
 
 ~~~~ abnf
 boolean-expr     = logical-or-expr
-logical-or-expr  = logical-and-expr *("||" logical-and-expr)
+logical-or-expr  = logical-and-expr *(S "||" S logical-and-expr)
                                                       ; disjunction
                                                       ; binds less tightly than conjunction
-logical-and-expr = basic-expr *("&&" basic-expr)      ; conjunction
+logical-and-expr = basic-expr *(S "&&" S basic-expr)  ; conjunction
                                                       ; binds more tightly than disjunction
 
-basic-expr   = exist-expr / paren-expr / (neg-op paren-expr) / relation-expr
+basic-expr   = exist-expr / 
```

Comment:
Style comment: I have tried to indicate ABNF operator precedence using parentheses.  Maybe not needed here because of the grouping into lines.